doc/api/perf_hooks.md

```diff
@@ -429,14 +429,15 @@ Returns a list of `PerformanceEntry` objects in chronological order
 with respect to `performanceEntry.startTime` whose `performanceEntry.entryType`
 is equal to `type`.
 
-## `perf_hooks.monitorEventLoopDelay([options])`
+## `perf_hooks.monitorEventLoopDelay()`
 <!-- YAML
 added: v11.10.0
 -->
 
 * `options` {Object}
   * `resolution` {number} The sampling rate in milliseconds. Must be greater
     than zero. **Default:** `10`.
+    **This property is deprecated and has no effect.**
```

Comment:
I would remove this from the documentation entirely and add a `changes:` entry to the YAML metadata, that should be sufficient imo.

Reply:
Ack.